public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -160,6 +194,46 @@ function Keyboard(menuUI) {
                         });
                     }
                     break;
+                    // "4" key
+                case 52:
+                    if (menuUI.noButton.hasClass('chosen') || menuUI.unsureButton.hasClass('chosen')) {
+                        if (svv.newValidateBeta) {
+                            menuUI.disagreeReasonTextBox.focus();
+                            menuUI.unsureReasonTextBox.focus();
+                        }
+                    }
+                    e.preventDefault();
+                    break;
+                // "1" key
+                case 49:
+                    if (menuUI.noButton.hasClass('chosen')) {
```

Comment:
Another weird thing is that `menuUI.noButton` will only have the "chosen" class on the new Validate. I'd prefer the we explicitly check that we're using the new Validate page before running this, rather than running it and it just happening to work out as expected because that CSS class doesn't exist on the other Validate page.